style.css

```diff
@@ -1 +1,190 @@
-/*  Add your styling here */
+/* Add your styling here */
+body{
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+}
+
+.logoup{
+    display: flex;
+    justify-content: center;
+    margin: auto;
+    width: 30%;
+}  
+
+/* cart icon */
+h3{
+    color: #ffffff;
+    float: right;
+    font-size: 16px;
+    padding: 0px 15px 0px 0px;
+
+}
+p{
+    display: flex;
+    justify-content: center;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    line-height: 1.5;
+    color:rgb(245, 245, 245);
+    background-color: rgb(17, 18, 19, 0.5);
+    padding: 0px 20px 0px 20px;
+}
+ul li{
+    float: left;
+    list-style: none;
+    position: relative;
+
+}
+a{
+   display: block; 
+   text-decoration: none;
+   color:white;
+   font-size: 22px;
+}
+ul li ul{
+    display: none;
+    position: absolute;
+    background-color:rgb(63, 8, 8,.8);
+    opacity: px;
```

Comment:
this is currently invalid css, opacity should always be a number between 0 an 1 and doesn't have a unit eg `opacity: 0.3`